Assets/Treasure/TDK/Runtime/Connect/Modals/TransitionModal.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Treasure
+{
+    // TODO reword in a way that prompts user to take action ("Sign into your account in the pop-up/browser"?)
```

Comment:
Left this comment because I'm not convinced on the wording of the transition screen, any suggestions are appreciated :)

Reply:
@lefarchi Thirdweb Connect modal on web says "Sign into your account in the pop-up" so I think that's fine to copy 🙂